test/blackbox/common/DDSBlackboxTestsPersistence.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Like it or not, it's 2020 already

Reply:
Sure, but the license could be issued in 2019. IDK